packages/blockchain-wallet-v4-frontend/src/services/CoinifyService/index.js

```diff
@@ -246,3 +247,22 @@ export const getReasonExplanation = (reason, time) => {
     default: return <FormattedMessage id='scenes.coinify.cannottradereason.unknown' defaultMessage='Trading is disabled.' />
   }
 }
+
+export const recurringTimeHelper = (sub) => {
+  let human = { 1: 'st', 2: 'nd', 3: 'rd', 21: 'st', 22: 'nd', 23: 'rd', 31: 'st' }
+  let days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+
+  let getTimespan = (sub) => {
+    let frequency = prop('frequency', sub)
```

Comment:
You can simplify that:
const freq = toLower(prop('frequency', sub))
By the way try to use const instead of let whenever you're defining something that will not need to be reassigned.